app/models/Organisation.js

```diff
@@ -60,7 +60,7 @@ const orgSchema = new Schema({
     }
   },
   contactInfo: {
-    emailId: {
+    communityEmail: {
```

Comment:
this should be just email as it is already inside community we dont need to specify for what this email id.